llvm/tools/llc/NewPMDriver.cpp

```diff
@@ -115,7 +115,7 @@ int llvm::compileModuleWithNewPM(
   MachineModuleInfo MMI(&LLVMTM);
 
   PassInstrumentationCallbacks PIC;
-  StandardInstrumentations SI(Context, Opt.DebugPM);
+  StandardInstrumentations SI(Context, Opt.DebugPM, !NoVerify);
```

Comment:
Command line option `verify-machineinstrs` is overlapped with `--disable-verify` when we use new pass manager.